app/assets/javascripts/assignments.js

```diff
@@ -25,7 +25,13 @@ $(document).ready(function() {
                             klass = 'btn-success';
                             break;
                     }
-                    $div.html('<a href="' + actions_map[mid].url + '" class="btn ' + klass + '" >' + actions_map[mid].action + '</a>')
+                    if ( actions_map[mid].action == 'Attempt' ){
```

Comment:
Better to fix the existing code smells if you happen to touch it. Can you fix it? Should be just a simple search and replace.